metadata/examples/complete_submission.json

```diff
@@ -9,6 +9,7 @@
     "size": 5884992
   },
   "submitter_is_author": true,
+  "submitter_contact_verified": true,
```

Comment:
I added this for now, since it is a required part of the submission process in the classic UI. But we should consider whether this really makes sense for the API when we circle back to user identification.

Reply:
Yes, this is good to keep for now.